bin/each-pkg

```diff
@@ -0,0 +1,112 @@
+#! /usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+const semver = require('semver');
+const program = require('commander');
+const { spawn, spawnSync } = require('child_process');
+
+/**
+ * Gets the package.json contents of each package located in the
+ * `./packages` subdirectory along with the package's path
```

Comment:
Ha, super stylistic, but can there be a space between this line and the `@ returns`?

Reply:
Added a space after all of the descriptions since that's what we're doing in the code so far